iphone/Classes/TiUIView.m

```diff
@@ -330,6 +330,7 @@ -(UIImage*)loadImage:(id)image
 		NSLog(@"[WARN] could not find image: %@",image);
 		return nil;
 	}
+    [TiUtils loadBackgroundImage:image forProxy:[self proxy]];
```

Comment:
Indentation. And I feel that this will cause caching problems, so I need some more time to think about the effect ob all our use-cases.